internal/ethapi/simulate_test.go

```diff
@@ -119,3 +121,18 @@ func TestSimulateSanitizeBlockOrder(t *testing.T) {
 func newInt(n int64) *hexutil.Big {
 	return (*hexutil.Big)(big.NewInt(n))
 }
+
+func TestHeaderPassing(t *testing.T) {
```

Comment:
Can probably delete this since it is only testing `procHeader`?

Reply:
Yeah that was just a scratch pad kind of test :) removed